java/java.hints/src/org/netbeans/modules/java/hints/errors/Utilities.java

```diff
@@ -3114,16 +3114,30 @@ public Void visitClass(ClassTree node, Void p) {
     public static boolean isCompatibleWithSwitchExpression(SwitchTree st) {
         boolean firstCase = true;
         Name leftTreeName = null;
-        for (CaseTree ct : st.getCases()) {
+        int CaseCount =0;
```

Comment:
Please correct source format and preferably var name also, start with lowercase letter.

Reply:
corrected in last commit